site/en/docs/extensions/mv3/permission_warnings/index.md

```diff
@@ -230,39 +211,44 @@ the steps in [Viewing Warnings][27].
     </tr>
     <tr id="contentSettings">
       <td><code>"contentSettings"</code></td>
-      <td>Grants your extension access to the <a href="/docs/extensions/reference/contentSettings">chrome.contentSettings</a> API.</td>
+      <td>Grants the extension access to the <a href="/docs/extensions/reference/contentSettings">chrome.contentSettings</a> API.</td>
       <td><strong>Change your settings that control websites' access to features such as cookies, JavaScript, plugins,
           geolocation, microphone, camera etc.</strong></td>
     </tr>
     <tr id="debugger">
       <td><code>"debugger"</code></td>
-      <td>Grants your extension access to the <a href="/docs/extensions/reference/debugger">chrome.debugger</a> API.</td>
+      <td>Grants the extension access to the <a href="/docs/extensions/reference/debugger">chrome.debugger</a> API.</td>
       <td>
         <ul>
           <li><strong>Access the page debugger backend</strong></li>
-          <li><strong>Read and change all your data on the websites you visit</strong></li>
+          <li><strong>Read and change all your data on all websites</strong></li>
```

Comment:
Is this the current string used in Chrome?

Reply:
Yes, I checked 👍